samples/nrf9160/aws_fota/Kconfig

```diff
@@ -64,6 +70,10 @@ config MQTT_BROKER_HOSTNAME
 config MQTT_BROKER_PORT
 	int
 	default 8883
+
+config NRF_CLOUD_CLIENT_ID_PREFIX
+	string "Prefix used when constructing the MQTT client ID from the IMEI"
+	default "nrf-"
```

Comment:
Why is this defined here, when it is also here:
https://github.com/nrfconnect/sdk-nrf/blob/14d54d4038e6c30202b61057a4aed0205f8a0d43/subsys/net/lib/nrf_cloud/Kconfig#L40-L42

Reply:
Because `aws_fota` can be used without the `nrf_cloud` module, in that case the `aws_fota` library still follows the nRF Connect for Cloud naming scheme, so it makes sense to have this value here with the same name. Note that `aws_fota` has a `USE_NRF_CLOUD` switch.

Reply:
I don't see how this explains why there cannot be a single `CLOUD_CLIENT_ID_PREFIX` setting.
If you use it with the `nrf_cloud`, then this value can default to `nrf-`, but if you use it with `aws_fota` then it can have another default value.

I don't see that both `CLOUD_CLIENT_ID_PREFIX` and `NRF_CLOUD_CLIENT_ID_PREFIX` are used at the same time in the same project, but I could be wrong on this ?

Reply:
No, that's right. It makes sense to use the same name, because it used for the same purpose.
I would rather call it `NRF_CLOUD_CLIENT_ID_PREFIX` because it is needed for a specific design decision within nRF Connect for Cloud and only useful for devices which connect there.

I will replace `CLOUD_CLIENT_ID_PREFIX` occurrences.